configure.ac

```
@@ -48,7 +48,8 @@ AC_PROG_CXX
 AM_PROG_CC_C_O
 AC_ISC_POSIX
 AC_HEADER_STDC
-IT_PROG_INTLTOOL([0.35.0])
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION([0.19.8])
```

I suggest also using `AM_GNU_GETTEXT_REQUIRE_VERSION`:
```suggestion
AM_GNU_GETTEXT_VERSION([0.19.8])
m4_ifdef([AM_GNU_GETTEXT_REQUIRE_VERSION],[
AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.8])
])
```
This means to pull in the latest gettext infrastructure >= 0.19.8 (if available), not just 0.19.8.